unit_test/batched/Test_Batched.hpp

```diff
@@ -77,9 +77,11 @@
 
 
 // Vector Kernels
+#if !defined(KOKKOSKERNELS_CUDA_BATCHED_TESTS)
```

Comment:
What errors are you seeing without this `ifdef`?

Reply:
These tests include the following static assertion:
```
   static_assert(Kokkos::Impl::SpaceAccessibility<DeviceType,Kokkos::HostSpace >::accessible,
		"vector datatype is only tested on host space");
```
which causes problems if building with a `Cuda` device...